Extensions/LoadableExtensionTemplate.json

```diff
@@ -0,0 +1,9 @@
+{
+  "$schema": "https://raw.githubusercontent.com/Slicer/Slicer/main/Schemas/slicer-extension-catalog-item-schema-v1.0.0.json#",
+  "build_dependencies": [],
+  "build_subdirectory": ".",
+  "category": "Examples",
+  "scm_type": "local",
+  "scm_revision": "NA",
```

Comment:
```suggestion
  "scm_revision": "",
```

Reply:
That would indeed be much cleaner and is now the opportunity.